testbed/core/migrations/0001_initial.py

```diff
@@ -1,177 +1,122 @@
-# Generated by Django 5.1.3 on 2024-12-04 23:50
+# Generated by Django 5.1.3 on 2025-05-06 04:49
```

Comment:
I assume you know this requires wiping the DB everywhere that the project has already been migrated/seeded -- that's fine at this stage

Reply:
Yes, I took that into consideration.